src/etc/inc/openvpn.inc

```diff
@@ -447,6 +447,16 @@ function openvpn_get_digestlist() {
 	return $digests;
 }
 
+function openvpn_get_keydirlist() {
+	$keydirs = array(
+		'-1' => gettext('Use default keydir direction'),
```

Comment:
an other is to remove this and for the client use the direction 1 as default (as it is now)
Eventually for the server it will be defaulted to 0. So we can remove this completely ?